### PR TITLE
en: update error handling descriptions (#265)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #265 to release-1.0

---

### What is changed, added or deleted? (Required)

- Add `workaround` for the error system
- Some minor updates

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: #243
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: en/TOC.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 ## TOC
 
+<<<<<<< HEAD
```
Comment (TomShawn) [inline en/TOC.md:8]: @lilin90 Please resolve the conflict, thanks!

File: en/TOC.md [hunk 2]
```diff
@@ -29,6 +30,23 @@
...
+    - [Block and Allow Lists](key-features.md#block-and-allow-table-lists)
+    - [Binlog Event Filter](key-features.md#binlog-event-filter)
+    - [Replication Delay Monitoring](key-features.md#replication-delay-monitoring)
+    - [Online DDL Scheme](feature-online-ddl-scheme.md)
+    - [Merge and Replicate Data from Sharded Tables](feature-shard-merge.md)
++ [Usage Scenarios](scenarios.md)
++ Quick Start
+  - [Deploy a DM Cluster](quick-start-with-dm.md)
+  - [Create Data Migration Task](quick-start-create-task.md)
+>>>>>>> 5492b9a... en: update error handling descriptions (#265)
```
Comment (lilin90) [inline en/TOC.md:49]:
```suggestion
```

File: en/TOC.md (hunk 1 above)
Comment (lilin90) [inline en/TOC.md:8]:
```suggestion
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**